<launch>

  <node pkg = "necst_ros3"
  	name = "dome_control"
	type = "dome_control.py">
    <param name="name_topic_from" value="cpz2724_rsw2_di11" />
  </node>

  <node pkg = "necst_ros3"
  	name = "dome_emergency"
	type = "dome_emergency.py">
    <param name="name_topic_from" value="cpz2724_rsw2_di1" />  #!!! dio ch need to be checked
  </node>

  <node pkg = "necst_ros3"
  	name = "dome_door"
	type = "dome_door.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_cmd"
	type = "dome_door_cmd.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_watcher_finalize_move"
	type = "dome_door_watcher_finalize_move.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_left_drive"
	type = "dome_door_left_drive.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_left_position"
	type = "dome_door_left_position.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_left_drive"
	type = "dome_door_left_drive.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_left_position"
	type = "dome_door_left_position.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_right_drive"
	type = "dome_door_right_drive.py" />

  <node pkg = "necst_ros3"
  	name = "dome_door_right_position"
	type = "dome_door_right_position.py" />

</launch>